pgwire/src/main/antlr/PgArray.g4

```diff
@@ -0,0 +1,55 @@
+grammar PgArray;
+
+array
```

Comment:
maybe a comment here describing what format this parser will parse could be helpful

Reply:
👍 i just added some examples. i didn't describe the format of the grammar, coz the it contains only two main rules, it should be relatively to see what is going on.

Reply:
👍 totally fine like that, thx!